canvas_sdk/tests/questionnaires/test_utils.py

```diff
@@ -59,7 +59,8 @@ def test_from_yaml_forbidden_questionnaire(
 
 def test_from_yaml_non_plugin_caller() -> None:
     """Test that the from_yaml function returns None when called outside a plugin."""
-    assert questionnaire_from_yaml("questionnaires/example_questionnaire.yml") is None
+    with pytest.raises(RuntimeError):
+        questionnaire_from_yaml("questionnaires/example_questionnaire.yml")
```

Comment:
@aduane the test you asked for is already here :)

Reply:
You know I saw this and for some reason didn't register it because it wasn't a test on the `get_cache` method.